frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -93,9 +96,11 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
             )}
             <dt>Host Addresses</dt>
             <dd>
-              Management: {getHostBMCAddress(host)}
-              <br />
-              NICs: {ips}
+              <ul className="metal3-bare-metal-host-details__host-addresses-list">
```

Comment:
Using list is definitely better, however I'd favour being consistent with other detail pages properties and use `span`s + `br`s. It would be good to define a common css classes for detail page property items list (e.g. `detail-page-property-items-list`, `detail-page-property-items-list__title`, `detail-page-property-items-list__item`) with similar styling which is used in `NodeIPList` component (used in node and machine detail pages).

Reply:
Why not use it then? List is much more semantically closer to this usage than br. I could change the other fields in this PR as well.

I guess for special cases yes. If we want to make it generic, I think it should rather go to https://patternfly-react.surge.sh/patternfly-4/documentation/react/components/list as it could be useful in broader sense.

Reply:
OK, could you please update all similar properties in BMH detail to use lists instead (using more generic css class names)?

Using List and ListItem from PF would be nice, however creating a wrapper component which tweaks the styles would be required.

Reply:
fixed

I meant, that it might be nice to have such a style switch inside the patternfly